docs/user_interface/how_to_deploy_ui.rst

```diff
@@ -0,0 +1,109 @@
+
+With your conda environment active, install the proper build requirements. Navigate to ``<pareto-ui>/backend`` and run the following command::
+
+    pip install -r requirements-build.txt
+
+
+.. _windows_application_build:
+
+**2. Build the application**
+
+To build the application installer, navigate to ``<pareto-ui>/electron`` and run the following command::
+
+    npm run dist:win
+
+The application build handles 3 substeps - building the backend, building the frontned, and building the electron package. 
+
+
+.. _windows_code_sign:
+
+**3. Code sign the application**
+
+For a complete guide to code signing electron applications, see their `code signing guide <https://www.electronjs.org/docs/latest/tutorial/code-signing>`_.
+
+The general steps required to obtaining and implementing a code signing certificate are as follows:
+
+* Purchase a code signing certificate. 
+
+    * This can be a standard certificate or an extended validation (EV) certificate
+    * For a list of certificate authorities, check out `Microsoft's documentation on managing code signing certificates <https://learn.microsoft.com/en-us/windows-hardware/drivers/dashboard/code-signing-cert-manage#get-or-renew-a-code-signing-certificate>`_
+
+* Implement the certificate key
+
+    * Using a hardware device such as a USB token OR
+    * Using an HSM or cloud hosted vault such as Azure Key Vault
+
+* Sign the Electron application
+
+    * with a CLI tool such as Microsoft SignTool if using a USB token
+    * with a CLI tool such as AzureSignTool if using Azure Key Vault
+
+.. _mac_guide_start:
+
+MacOS (ARM64)
+-------------
+
+.. _mac_build_requirements:
+
+**1. Install the build requirements**
+
+With your conda environment active, install the proper build requirements. Navigate to ``<pareto-ui>/backend`` and run the following command::
+
+    pip install -r requirements-build.txt
+
+
+.. _mac_application_build:
+
+**2. Build the application**
+
+To build the application installer, navigate to ``<pareto-ui>/electron`` and run the following command::
+
+    npm run dist:mac
+
+The application build handles 3 substeps - building the backend, building the frontned, and building the electron package. 
+
+
+.. _mac_code_sign:
+
+**3. Code sign the application**
+
+For a complete guide to code signing electron MacOS applications, see the `Mac App Store Submission Guide <https://www.electronjs.org/docs/latest/tutorial/mac-app-store-submission-guide>`_.
+
+
+The general steps required to obtaining and implementing an Apple code signing certificate are as follows:
+
+* Create an `Apple developer account <https://developer.apple.com/>`_.
+* Attain a `Developer ID Application certificate <https://developer.apple.com/help/account/create-certificates/create-developer-id-certificates/>`_ and add it to your Keychain
+* Create a ``.env`` file in the ``<pareto-ui>/electron`` directory with the following variables:
+
+    * APPLE_ID
+    * APPLE_ID_PASSWORD
+    * TEAM_ID*
```

Comment:
You could replace the asterisk with an inline link down to the note below see https://docutils.sourceforge.io/docs/user/rst/quickref.html#internal-hyperlink-targets for syntax

Reply:
@MichaelPesce I would bring back the asterisks here. The inline link is fine but I think having the asterisks was helpful.

Reply:
And maybe put an inline link on TEAM_ID in addition to NOTARIZE_TOOL.